webapp/client/src/components/FormFieldTaxNumber.test.js

```diff
@@ -31,29 +31,29 @@ describe("FormFieldTaxNumber is split", () => {
   });
 
   describe("When typing 12 characters into each input", () => {
-    const input_characters = "123456789012";
+    const inputCharacters = "123456789012";
 
     beforeEach(() => {
-      userEvent.type(screen.getAllByRole("textbox")[0], input_characters);
-      userEvent.type(screen.getAllByRole("textbox")[1], input_characters);
-      userEvent.type(screen.getAllByRole("textbox")[2], input_characters);
+      userEvent.type(screen.getAllByRole("textbox")[0], inputCharacters);
+      userEvent.type(screen.getAllByRole("textbox")[1], inputCharacters);
+      userEvent.type(screen.getAllByRole("textbox")[2], inputCharacters);
     });
 
-    it("First input should only contain first 3 characters", () => {
+    it("First input should only contain first 2 characters", () => {
       expect(screen.getAllByRole("textbox")[0]).toHaveValue(
-        input_characters.slice(0, 3)
+        inputCharacters.slice(0, 2)
```

Comment:
I'd find these tests to be easier to comprehend (and also better documentation of the expected behaviour) if they were written with explicit values, like so:
`expect(screen.getAllByRole("textbox")[0]).toHaveValue("12")`

An example to go on my "DRY sometimes considered harmful in tests" list :)